### PR TITLE
chore(pt-br): replace interwiki macros with markdown links

### LINKED ISSUE #7287: Removal of `{{Interwiki}}` macro
Per https://github.com/mdn/mdn-community/discussions/165
Opening an issue in order to track this more easily with labels.

- [x]  @mdn/yari-content-es : ~188~ 185 files https://github.com/mdn/translated-content/pull/9818 and https://github.com/mdn/translated-content/pull/9819
- [x]  @mdn/yari-content-fr : 265 files https://github.com/mdn/translated-content/pull/7288, https://github.com/mdn/translated-content/pull/7288
- [x] @mdn/yari-content-ja : ~334~ 324 files #8735 
- [x] @mdn/yari-content-ko : 122 files https://github.com/mdn/translated-content/pull/7426
- [ ] @mdn/yari-content-pt-br : 104 files #13922
- [x] @mdn/yari-content-ru : 137 files #11511
- [x] @mdn/yari-content-zh : 
  - [x] `zh-CN` : 193 files https://github.com/mdn/translated-content/pull/7306
  - [x] `zh-TW` : 116 files https://github.com/mdn/translated-content/pull/7307

### PR DESCRIPTION
**Pull request by yin1999**
### Description

replace interwiki macros with markdown links

I've used a script to replace most of them:

```js
import * as fs from 'fs';
import * as path from 'path';

const root = 'files/pt-br/';

async function walk(dir, callback) {
	const files = fs.readdirSync(dir, { withFileTypes: true });
	for (const file of files) {
		const filepath = path.join(dir, file.name);
		if (file.isDirectory()) {
			walk(filepath, callback);
		} else if (file.name.endsWith('.md')) {
			await callback(filepath);
		}
	}
}

function safeUrl(url) {
	if (url.includes('(') || url.includes(')')) {
		return `<${url}>`;
	}
	return url;
}

async function resolve(filepath) {
	const content = fs.readFileSync(filepath, 'utf8');
	const re = /\{\{\s*interwiki\s*\(['"]wikipedia['"]\s*,\s*['"]([^'"]+)['"]\s*\)\s*\}\}/ig;
	// replace all occurrences
	const names = [];
	content.replace(re, (match, name) => {
		// check whether the wikipedia page exists
		names.push(name);
	});
	if (names.length == 0) {
		return;
	}
	const replaces = [];
	for (const name of names) {
		try {
			let lang = 'en'; // could replace this with 'pt' to resolve some "correct" interwiki macro calls
			let apiUrl = `https://${lang}.wikipedia.org/w/api.php?action=query&titles=${encodeURIComponent(name)}&prop=langlinks&lllimit=500&format=json` // could replace `pt` with `en` to get translations when not existed in `pt`
			const response = await fetch(apiUrl);
			if (!response.ok || response.status != 200) {
				throw new Error(`HTTP ${response.status}`);
			} else {
				const json = await response.json();
				const pages = json.query.pages;
				const page = pages[Object.keys(pages)[0]];
				if (page.hasOwnProperty('missing')) {
					throw new Error(`Page not found`);
				}
				let title = page.title;
				if (lang !== 'pt') {
					for (const langlink of (page.langlinks || [])) {
						if (langlink.lang == 'pt') {
							title = langlink['*'];
							lang = 'pt';
							break;
						}
					}
				}
				replaces.push([name, `https://${lang}.wikipedia.org/wiki/${title.replace(/ /g, '_')}`]);
			}
		} catch (e) {
			console.error(`Error resolving file ${filepath}: ${e.message}`);
			replaces.push([null, null]);
		}
	}
	const newContent = content.replace(re, (match) => {
		const [name, url] = replaces.shift();
		if (!name) {
			return match;
		}
		return `[${name}](${safeUrl(url)})`;
	})
	if (newContent == content) {
		return;
	}
	fs.writeFileSync(filepath, newContent);
}

await walk(root, resolve);
```

### Related issues and pull requests

Part of #7287

### CODE REVIEW

File: files/pt-br/glossary/ux/index.md [hunk 1]
```diff
@@ -12,4 +12,4 @@ O sistema pode ser qualquer produto ou aplicativo com o qual um usuário final d
 
 ### Conhecimento geral
 
-- {{Interwiki("wikipedia", "Experiência do Usuário")}} na Wikipédia
+- [Experiência do Usuário](https://pt.wikipedia.org/wiki/Experiência_do_Usuário) na Wikipédia
```
Comment (josielrocha) [inline files/pt-br/glossary/ux/index.md:15]:
```suggestion
- [Experiência do Usuário](https://pt.wikipedia.org/wiki/Experiência_do_usuário) na Wikipédia
```

File: files/pt-br/glossary/value/index.md [hunk 2]
```diff
@@ -10,4 +10,4 @@ No contexto de dados ou um objeto {{Glossary("Wrapper", "wrapper")}} em torno de
 
 ### Conhecimento geral
 
-- {{Interwiki("wikipedia", "Primitive wrapper class")}} on Wikipedia
+- [Primitive wrapper class](https://pt.wikipedia.org/wiki/Primitive_wrapper_class) on Wikipedia
```
Comment (josielrocha) [inline files/pt-br/glossary/value/index.md:13]: This URL does not exist in pt-BR

```suggestion
- [Primitive wrapper class](https://en.wikipedia.org/wiki/Primitive_wrapper_class_in_Java) on Wikipedia
```

File: files/pt-br/glossary/vendor_prefix/index.md [hunk 3]
```diff
@@ -35,4 +35,4 @@ Em propriedades e métodos, eles normalmente usam:
 
 ### Conhecimento geral
 
-- {{Interwiki("wikipedia", "CSS filter#Prefix_filters", "Vendor prefix")}} no Wikipedia
+- [Vendor prefix](https://pt.wikipedia.org/wiki/CSS_filter#Prefix_filters) no Wikipedia
```
Comment (josielrocha) [inline files/pt-br/glossary/vendor_prefix/index.md:38]: This URL does not exist in pt-BR

```suggestion
- [Vendor prefix](https://en.wikipedia.org/wiki/CSS_hack#Prefix_filters) no Wikipedia
```

File: files/pt-br/glossary/viewport/index.md [hunk 4]
```diff
@@ -10,7 +10,7 @@ A porção da viewport que está visível no momento é chamada de **visual view
 
 1. Conhecimentos gerais
 
-    1. {{Interwiki("wikipedia", "Viewport")}} na Wikipedia (em inglês)
+    1. [Viewport](https://pt.wikipedia.org/wiki/Viewport) na Wikipedia (em inglês)
```
Comment (josielrocha) [inline files/pt-br/glossary/viewport/index.md:13]: This URL does not exist in pt-BR

```suggestion
    1. [Viewport](https://en.wikipedia.org/wiki/Viewport) na Wikipedia (em inglês)
```

File: files/pt-br/glossary/websockets/index.md [hunk 5]
```diff
@@ -12,7 +12,7 @@ Qualquer aplicação de servidor ou cliente pode usar WebSocket, mas principalme
 
 ### General knowledge
 
-- {{Interwiki("wikipedia", "Websocket")}} on Wikipedia
+- [Websocket](https://pt.wikipedia.org/wiki/Websocket) on Wikipedia
```
Comment (josielrocha) [inline files/pt-br/glossary/websockets/index.md:15]:
```suggestion
- [Websocket](https://pt.wikipedia.org/wiki/WebSocket) on Wikipedia
```

File: files/pt-br/glossary/wrapper/index.md [hunk 6]
```diff
@@ -12,4 +12,4 @@ Por exemplo, as bibliotecas do SDK para AWS são exemplos de wrappers.
 
 ### Conhecimento Geral
 
-{{Interwiki("wikipedia", "Wrapper function")}} em Wikipedia
+[Wrapper function](https://pt.wikipedia.org/wiki/Wrapper_function) em Wikipedia
```
Comment (josielrocha) [inline files/pt-br/glossary/wrapper/index.md:15]: This URL does not exist in pt-BR

```suggestion
[Wrapper function](https://en.wikipedia.org/wiki/Wrapper_function) em Wikipedia
```

File: files/pt-br/orphaned/glossary/dynamic_programming_language/index.md [hunk 7]
```diff
@@ -14,4 +14,4 @@ Isso é o oposto das linguagens denominadas **linguagens de programação estát
 
 ### Conhecimento geral
 
-- {{Interwiki("wikipedia", "Linguagem de Programação Dinâmica")}} on Wikipedia
+- [Linguagem de Programação Dinâmica](https://pt.wikipedia.org/wiki/Linguagem_de_Programação_Dinâmica) on Wikipedia
```
Comment (josielrocha) [inline files/pt-br/orphaned/glossary/dynamic_programming_language/index.md:17]: This URL does not exist. Should we remove it?

File: files/pt-br/web/http/status/418/index.md [hunk 8]
```diff
@@ -25,4 +25,4 @@ O código de erro HTTP para o cliente **`418 I'm a teapot`** indica que o servid
 
 ## Veja também
 
-- {{interwiki("wikipedia", "Hyper Text Coffee Pot Control Protocol", "Wikipedia: Hyper Text Coffee Pot Control Protocol")}}
+- [Wikipedia: Hyper Text Coffee Pot Control Protocol](https://pt.wikipedia.org/wiki/Hyper_Text_Coffee_Pot_Control_Protocol)
```
Comment (josielrocha) [inline files/pt-br/web/http/status/418/index.md:28]: This URL does not exist in pt-BR

```suggestion
- [Wikipedia: Hyper Text Coffee Pot Control Protocol](https://en.wikipedia.org/wiki/Hyper_Text_Coffee_Pot_Control_Protocol)
```

File: files/pt-br/web/http/status/504/index.md [hunk 9]
```diff
@@ -7,7 +7,7 @@ slug: Web/HTTP/Status/504
 
 O código de resposta de erro HTTP **`504 Gateway Timeout`** indica que o servidor, enquanto atuando como gateway ou proxy, não conseguiu responder em tempo.
 
-Um {{interwiki("wikipedia", "Gateway_(telecommunications)", "Gateway")}} pode referir-se a diferentes elementos de rede e um erro 504 geralmente é algo que você não pode consertar, mas solicitar uma correção no servidor web ou proxy no qual você está tentando acessar.
+Um [Gateway](<https://pt.wikipedia.org/wiki/Gateway_(telecommunications)>) pode referir-se a diferentes elementos de rede e um erro 504 geralmente é algo que você não pode consertar, mas solicitar uma correção no servidor web ou proxy no qual você está tentando acessar.
```
Comment (josielrocha) [inline files/pt-br/web/http/status/504/index.md:10]:
```suggestion
Um [Gateway](<https://pt.wikipedia.org/wiki/Gateway>) pode referir-se a diferentes elementos de rede e um erro 504 geralmente é algo que você não pode consertar, mas solicitar uma correção no servidor web ou proxy no qual você está tentando acessar.
```

File: files/pt-br/web/http/status/502/index.md [hunk 10]
```diff
@@ -7,7 +7,7 @@ slug: Web/HTTP/Status/502
 
 O código de erro HTTP **`502 Bad Gateway`** retornado pelo servidor indica que ele, enquanto atuando como um servidor intermediário (_gateway_ ou _proxy_), recebeu uma resposta inválida do servidor para o qual a requisição foi encaminhada (_upstream server_).
 
-> **Nota:**Um {{interwiki("wikipedia", "Gateway_(telecommunications)", "<em>Gateway</em>")}} pode se referir a coisas diferentes na rede e um erro 502 geralmente não é algo que você possa corrigir, mas requer uma correção por parte do servidor web ou dos proxies através dos quais você está tentando obter acesso.
+> **Nota:**Um [<em>Gateway</em>](<https://pt.wikipedia.org/wiki/Gateway_(telecommunications)>) pode se referir a coisas diferentes na rede e um erro 502 geralmente não é algo que você possa corrigir, mas requer uma correção por parte do servidor web ou dos proxies através dos quais você está tentando obter acesso.
```
Comment (josielrocha) [inline files/pt-br/web/http/status/502/index.md:10]:
```suggestion
> **Nota:**Um [<em>Gateway</em>](<https://pt.wikipedia.org/wiki/Gateway>) pode se referir a coisas diferentes na rede e um erro 502 geralmente não é algo que você possa corrigir, mas requer uma correção por parte do servidor web ou dos proxies através dos quais você está tentando obter acesso.
```

**Review by josielrocha — COMMENTED** [review]
Thank you very much for your help @yin1999!

I made some suggestions. Could you please review them?

File: files/pt-br/orphaned/glossary/dynamic_programming_language/index.md (hunk 7 above)
Comment (yin1999) [reply files/pt-br/orphaned/glossary/dynamic_programming_language/index.md:17]: Yes, we could remove this (actually, this document [should be removed](https://github.com/mdn/translated-content/blob/main/docs/orphaned_and_conflicting.md#orphaned-pages))

**Comment by yin1999** [comment]
> I made some suggestions. Could you please review them?

Sorry about this. I didn't check all the urls. I've made a mistake in the script (the api always return 200 even if the page is not existed, I've updated the script to resolve the problem. Hi @josielrocha, could you have a review on those changes :)

**Review by josielrocha — APPROVED** [review]

**Pull request merged**